openpype/pipeline/publish/lib.py

```diff
@@ -465,7 +465,7 @@ def apply_plugin_settings_automatically(plugin, settings, logger=None):
     for option, value in settings.items():
         if logger:
             logger.debug("Plugin {} - Attr: {} -> {}".format(
-                option, value, plugin.__name__
+                plugin.__name__, option, value
             ))
```

Comment:
@iLLiCiTiT might be even better like this:
```python
            logger.debug("Plugin %s - Attr: %s -> %s", 
                                 plugin.__name__, option, value)
```
For cases where this isn't handled by handler?

Reply:
Did you try it? I'm ok with it, if it works with `value` which is `dict`.

Reply:
Seems fine in maya with outputs like this:
```
#   - { filter_pyblish_plugins }: [ Plugin ProcessSubmittedJobOnFarm - Attr: enabled -> True ] 
#   - { filter_pyblish_plugins }: [ Plugin ProcessSubmittedJobOnFarm - Attr: deadline_department ->  ] 
#   - { filter_pyblish_plugins }: [ Plugin ProcessSubmittedJobOnFarm - Attr: deadline_pool ->  ] 
#   - { filter_pyblish_plugins }: [ Plugin ProcessSubmittedJobOnFarm - Attr: deadline_group ->  ] 
#   - { filter_pyblish_plugins }: [ Plugin ProcessSubmittedJobOnFarm - Attr: deadline_chunk_size -> 1 ] 
#   - { filter_pyblish_plugins }: [ Plugin ProcessSubmittedJobOnFarm - Attr: deadline_priority -> 50 ] 
#   - { filter_pyblish_plugins }: [ Plugin ProcessSubmittedJobOnFarm - Attr: publishing_script ->  ] 
#   - { filter_pyblish_plugins }: [ Plugin ProcessSubmittedJobOnFarm - Attr: skip_integration_repre_list -> [] ] 
#   - { filter_pyblish_plugins }: [ Plugin ProcessSubmittedJobOnFarm - Attr: aov_filter -> {'maya': ['.*([Bb]eauty).*'], 'aftereffects': ['.*'], 'celaction': ['.*'], 'harmony': ['.*'], 'max': ['.*'], 'fusion': ['.*']} ] 
#   - { filter_pyblish_plugins }: [ Plugin CleanUp - Attr: paterns -> [] ] 
#   - { filter_pyblish_plugins }: [ Plugin CleanUp - Attr: remove_temp_renders -> False ] 
#   - { filter_pyblish_plugins }: [ Plugin CleanUpFarm - Attr: enabled -> False ] 
```